### ISSUE TITLE
Bugged High Elf zone quest "Gathering Intel"

### ISSUE THREAD

**Issue by Floralys23** [slowtorta/turtlewow-bug-tracker#405]
**Please, provide a clear description of what the bug is:**
The quest "Gathering Intel" in the High Elf starting zone doesn't trigger quest completion upon scouting the area around the Shadowforge Dwarves' camp. I have also tried going inside the camp, but to no avail.

**Steps to reproduce the behavior with as much detail as possible:**
1. make a High Elf character 
2. proceed with levelling
3. accept the quest Gathering Intel
4. try to complete the quest Gathering Intel by exploring the area around the Shadowforge camp

**IDs of affected creatures, items, quests or spells with a link to the relevant page:**
Quest: https://database.turtle-wow.org/?quest=80204

**Expected behavior. Describe how it should work:**
The quest should be completed when you approach the camp.
![WoWScrnShot_110221_144004](https://user-images.githubusercontent.com/93597799/139865346-be027ffe-2c28-4755-8053-4727b986829b.jpg)

![WoWScrnShot_110221_143950](https://user-images.githubusercontent.com/93597799/139864939-c079dbb8-38c9-4cba-81c0-102feb7e8fca.jpg)
![WoWScrnShot_110221_143958](https://user-images.githubusercontent.com/93597799/139865004-3e5e3a6b-9e27-4489-bbaa-b26f5351f758.jpg)

**Comment by slowtorta** [comment]
Hello there. Just tested it and it worked fine. Try to not actually enter the camp but scout area around it (as it's stated in the quest text).

**Comment by Floralys23** [comment]
> 
> 
> Hello there. Just tested it and it worked fine. Try to not actually enter the camp but scout area around it (as it's stated in the quest text).


I HAVE scouted the area around the camp earlier. I only entered the camp because I wasn't able to trigger it outside.

I've just tried it again (character name Anglesey) and it still doesn't work. Moreover, I've even completed the quest that requires you to kill the Shadowforge Priest and free the high elven children ( https://database.turtle-wow.org/?quest=80207 ) - without ever triggering the completion of Gathering Intel. Something is definitely not working as intended here. 

I would also like to point out that I wasn't the only one having this issue. Several other players, HC and non-HC alike, whom I have encountered while trying to complete the quest, have not been able to finish the quest.
